recipes/libdxfrw/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.0.1":
```

Comment:
The original project does not offer such version: https://sourceforge.net/projects/libdxfrw

Use the upstream version. You can this recipe as reference: https://github.com/conan-io/conan-center-index/blob/64deb844f5fbee9dd80325dd747d68e2a248c0ea/recipes/libconfuse/all/conanfile.py#L64

Reply:
The original project on SourceForge is not maintained anymore : https://sourceforge.net/p/libdxfrw/discussion/general/thread/a71f71d3/#03ec/c469

This is the version we use in LibreCAD, which includes bug and security fixes.